src/instance/minecraft/commands/BitchesCommand.ts

```diff
@@ -6,7 +6,10 @@
 import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
 
 export default {
+  name: 'Bitches',
   triggers: ['bitch', 'bitches', 'b'],
+  description: 'Returns how many "b\'s" a user has',
+  example: `b Hxqz`,
```

Comment:
All examples start with the command prefix itself. make it dynamic instead.

```suggestion
  example: `Hxqz`,
```

And in CommandManager you do 
```ts
await minecraftInstance.send(
        `/gc ${command.name}: ${command.description} (${minecraftInstance.config.commandPrefix}${command.triggers[0]} ${command.example})`
      )
```

Reply:
I dont like this, some example params would become `null` or something like`%s` only